robotswerve.py

```diff
@@ -25,6 +25,8 @@
 from commands.togglefielddrive import ToggleFieldDrive
 from commands.resetfielddrive import ResetFieldDrive
 
+from auto import SparkyShoot
+
 import math
```

Comment:
```suggestion
```

Linter flagged this as unused https://github.com/Raptacon/Robot-2024/actions/runs/8073244520/job/22056564167?pr=59